New package: RobertoConcepcion.MaxLauncher version 1.31.0.0

Dragon1573 opened the pull request:
## What's changed

- Resolve https://github.com/microsoft/winget-pkgs/issues/117453

## Check

- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.5 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.5.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/122410)

Dragon1573 committed:
New package: RobertoConcepcion.MaxLauncher version 1.31.0.0

Dragon1573 commented:
Manifest is valid while testing in Windows Sandbox. Does it forbidden to download appications from SourceForge via `winget.exe`?

<details><summary>Test logs</summary>

```text
PS (Host)> .\Tools\SandboxTest.ps1 .\manifests\r\RobertoConcepcion\MaxLauncher\1.31.0.0\
--> Validating Manifest
清单验证成功。

--> Checking dependencies
--> Starting Windows Sandbox, and:
    - Mounting the following directories:
      - C:\Users\Dragon1573\AppData\Local\Temp\SandboxTest as read-only
      - D:\Repository\winget-pkgs as read-and-write
    - Installing WinGet
    - Configuring Winget
    - Installing the Manifest 1.31.0.0
    - Refreshing environment variables
    - Comparing ARP Entries
```

```text
--> Installing WinGet
--> Disabling safety warning when running installer
Tip: you can type 'Update-EnvironmentVariables' to update your environment variables, such as after installing a new software.

--> Configuring Winget
Enabled admin setting 'LocalManifestFiles'.
Enabled admin setting 'LocalArchiveMalwareScanOverride'.


--> Installing the Manifest 1.31.0.0

Found MaxLauncher [RobertoConcepcion.MaxLauncher] Version 1.31.0.0
This application is licensed to you by its owner.
Microsoft is not responsible for, nor does it grant any licenses to, third-party packages.
Downloading https://sourceforge.net/projects/maxlauncher/files/MaxLauncher/1.31.0.0/maxlauncher_1.31.0.0_setup.exe/download
  ██████████████████████████████  1.79 MB / 1.79 MB
Successfully verified installer hash
Starting package install...
Successfully installed

--> Refreshing environment variables

--> Comparing ARP Entries

DisplayName                  DisplayVersion Publisher          ProductCode                                Scope
-----------                  -------------- ---------          -----------                                -----
MaxLauncher version 1.31.0.0 1.31.0.0       Roberto Concepcion {D887E7A5-7906-4A0B-9E16-791BD8F2FD2F}_is1 Machine
```

</details>

<details><summary>Screenshots</summary>

![image](https://github.com/microsoft/winget-pkgs/assets/49941141/1162c999-0d2b-4cd6-8998-8ea9b15a39cf)
![image](https://github.com/microsoft/winget-pkgs/assets/49941141/7bf68509-1c46-45d6-a0e2-b512fdd43786)
![image](https://github.com/microsoft/winget-pkgs/assets/49941141/c3cad4fa-26e7-4ee2-92f6-00f6c1af593d)

</details>

stephengillie commented on manifests/r/RobertoConcepcion/MaxLauncher/1.31.0.0/RobertoConcepcion.MaxLauncher.locale.en-US.yaml:
```diff
@@ -0,0 +1,22 @@
+PackageVersion: 1.31.0.0
+PackageLocale: en-US
+Publisher: Roberto Concepcion
+PublisherUrl: https://sourceforge.net/
+PublisherSupportUrl: https://sourceforge.net/support
+PrivacyUrl: https://slashdotmedia.com/privacy-statement/
+Author: madproton
+PackageName: MaxLauncher
+PackageUrl: https://maxlauncher.sourceforge.io/
+License: Microsoft Reciprocal License
+LicenseUrl: https://maxlauncher.sourceforge.io/license.html
```
```suggestion
PublisherUrl: https://maxlauncher.sourceforge.io/
PublisherSupportUrl: https://maxlauncher.sourceforge.io/support.html
# PrivacyUrl: 
Author: madproton
PackageName: MaxLauncher
PackageUrl: https://maxlauncher.sourceforge.io/
License: Microsoft Reciprocal License
LicenseUrl: https://maxlauncher.sourceforge.io/license.html
```

stephengillie commented on manifests/r/RobertoConcepcion/MaxLauncher/1.31.0.0/RobertoConcepcion.MaxLauncher.locale.en-US.yaml:
```diff
@@ -0,0 +1,22 @@
+PackageVersion: 1.31.0.0
+PackageLocale: en-US
+Publisher: Roberto Concepcion
+PublisherUrl: https://sourceforge.net/
+PublisherSupportUrl: https://sourceforge.net/support
+PrivacyUrl: https://slashdotmedia.com/privacy-statement/
```
```suggestion
PublisherUrl: https://sourceforge.net/projects/maxlauncher/
PublisherSupportUrl: https://maxlauncher.sourceforge.io/support.html
# PrivacyUrl: 
```

stephengillie commented:
> Manifest is valid while testing in Windows Sandbox. Does it forbidden to download appications from SourceForge via `winget.exe`?
> 
> Test logs
> Screenshots

Hi @Dragon1573,

Applications from this host aren't forbidden, but require manual approval, because this site is used both for direct ISV hosting and also 3rd party rehosting. 
Also, in this instance, the hosting site's URLs (and another site's URL) were used instead of the ISV's.

Dragon1573 committed:
Fixup: Update publisher informations

- Switch publisher from SourceForge to MaxLauncher itself
- Remove privacy URL of the manifest

Co-authored-by: Stephen Gillie <114952565+stephengillie@users.noreply.github.com>

Dragon1573 committed:
Fixup: Change installer URL

- This URL is copied from the browser "Download" info

Signed-off-by: Dragon1573 <49941141+Dragon1573@users.noreply.github.com>

Dragon1573 committed:
Update: Locale informations

- Remove privacy URL
- Change copyright character
- Format full descriptions for readability

Signed-off-by: Dragon1573 <49941141+Dragon1573@users.noreply.github.com>

stephengillie commented:
@wingetbot waivers Add Validation-Unapproved-URL
